src/components/textAreas/textArea/textArea.tsx

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+import { InputContainer, type IInputComponentProps } from '../../input';
+import { useInputProps } from '../../input/hooks';
+
+export interface ITextAreaProps extends IInputComponentProps<HTMLTextAreaElement> {}
+
+export const TextArea: React.FC<ITextAreaProps> = (props: ITextAreaProps) => {
+    const { containerProps, inputProps } = useInputProps(props);
+    const { className: inputClassName, ...otherInputProps } = inputProps;
+
+    return (
+        <InputContainer wrapperClassName="grow" {...containerProps}>
+            <textarea type="text" className={classNames('min-h-[144px]', inputClassName)} {...otherInputProps} />
```

Comment:
Thoughts about adding a slight margin so that both of the diagonal lines indicating that the textarea is resizable can show

Reply:
It's a good idea, but it does not look good when having a scrollbar :/
<img width="468" alt="Screenshot 2024-01-31 at 15 36 37" src="https://github.com/aragon/ods/assets/19923822/3a17f47c-6fe3-4ad8-aff7-e123bbca9f49">
I'd suggest to keep this as is for now

Reply:
Yeah, let's get confirmation because by default only part of it is showing.

![image](https://github.com/aragon/ods/assets/16927999/838bbe60-0a92-429c-8bf2-e9d7cf5d6125)